src/modelmatrix.jl

```diff
@@ -1,6 +1,22 @@
 
 typealias AbstractFloatMatrix{T<:AbstractFloat} AbstractMatrix{T}
 
+"""
+Convert a `ModelFrame` into a numeric matrix suitable for modeling
```

Comment:
I was working off DataFrames.jl, which gives a short summary, then lists constructors and their arguments. I'm not particularly wedded to this way but I'm also not aware of any other standard besides that